config.json

```diff
@@ -40,7 +40,7 @@
       "description": "MsBuild logging options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/l:BinClashLogger,Tools/net46/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log"
```

Comment:
@ericstj Nevermind.  I think we figured it out.  coreclr's build-test.cmd is overriding this setting to disable binclashlogger.  Ugh.